man/systemd.exec.xml

```diff
@@ -2222,6 +2226,36 @@ StandardInputData=SWNrIHNpdHplIGRhIHVuJyBlc3NlIEtsb3BzLAp1ZmYgZWVtYWwga2xvcHAncy
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>LogNamespace=</varname></term>
+
+        <listitem><para>Run the unit's processes in the specified journal namespace. Expects a short
+        user-defined string identifying the namespace. If not used the processes of the service are run in
```

Comment:
Is there a possibility of one user specifying a `namespace` that conflicts with another user's `namespace`?

Reply:
the journal is a system service, always has been, and the instances of it are too. I think people should grok that per system you can have only one service instance under one name. I mean, that design decision is not new to the journal namespace concept really, it's true for any kind of service instance, and we make very clear, in one of the first sentences of the journal namespace description that this is implemented via unit namespaces.